webapp/src/Service/ScoreboardService.php

```diff
@@ -464,19 +488,21 @@ public function calculateScoreRow(
             'submissionsRestricted' => $submissionsJury,
             'pendingRestricted' => $pendingJury,
             'solvetimeRestricted' => (int)$timeJury,
+            'runtimeRestricted' => $runtimeJury === PHP_INT_MAX ? 0 : $runtimeJury,
```

Comment:
Maybe better to use a sentinel value of `-1` or just `null`?

Reply:
I use PHP_INT_MAX because it is a neutral element for `min` and I can write 
```php
$runtimeJury = min($runtimeJury, $runtime);
...
$runtimePubl = min($runtimePubl, $runtime);
```
instead of
```php
if (is_null($runtimeJury) || $runtimeJury > $runtime) {
    $runtimeJury = $runtime;
}
...
if (is_null($runtimePubl) || $runtimePubl > $runtime) {
    $runtimePubl = $runtime;
}
```

Reply:
To clarify: I wasn't referring to the `PHP_INT_MAX` (I agree with that), but the `0` after the `?`.

Reply:
Ah, sorry. 
Idk if null would be good there. I used 0 to keep the `runtime_...` fields of ScoreCache consistent with the `solvetime_...` fields and those use 0 for unsolved problems and have `nullable=false` in the DB.

Reply:
Ah, ok, sorry: I wasn't aware that these are not nullable and already follow this behaviour. Then please ignore this.